.applier/templates/build.yml

```diff
@@ -36,6 +36,8 @@ objects:
         from:
```

Comment:
Same. maybe a bit different. Is this a generic build file? If there is nothing in redhat-cop should we add one. Seems like pullSecret might be unique.